tracing-subscriber/src/filter/env/builder.rs

```diff
@@ -0,0 +1,316 @@
+use super::{
+    directive::{self, Directive},
+    EnvFilter, FromEnvError,
+};
+use crate::sync::RwLock;
+use std::env;
+use thread_local::ThreadLocal;
+use tracing::level_filters::STATIC_MAX_LEVEL;
+
+/// A [builder] for constructing new [`EnvFilter`]s.
+///
+/// [builder]: https://rust-unofficial.github.io/patterns/patterns/creational/builder.html
```

Comment:
I didn't think this needed to be linked, but... I guess it's handy? idk.

Reply:
i definitely didn't think it was at all necessary, but i thought it might be worth including for newcomers?

Reply:
yeah, i'd keep it—just haven't seen that be linked before, but given how popular `tracing-subscriber` is getting, it's a good idea.